src/gen_rules.ml

```diff
@@ -279,29 +279,34 @@ module Gen(P : Install_rules.Params) = struct
      +-----------------------------------------------------------------+ *)
 
   let guess_modules ~dir ~files =
+    let make_module syntax base fn =
+      (Module.Name.of_string base,
+       Module.File.make syntax (Path.relative dir fn))
+    in
     let impl_files, intf_files =
       String.Set.to_list files
       |> List.filter_partition_map ~f:(fun fn ->
         (* we aren't using Filename.extension because we want to handle
            filenames such as foo.cppo.ml *)
         match String.lsplit2 fn ~on:'.' with
-        | Some (_, "ml") -> Left { Module.File.syntax=OCaml ; name=fn }
-        | Some (_, "re") -> Left { Module.File.syntax=Reason ; name=fn }
-        | Some (_, "mli") -> Right { Module.File.syntax=OCaml ; name=fn }
-        | Some (_, "rei") -> Right { Module.File.syntax=Reason ; name=fn }
+        | Some (s, "ml" ) -> Left  (make_module OCaml  s fn)
```

Comment:
Minor point but I don't think it's a good idea to align parens here. It requires some effort to stay aligned and the benefit in doing so is pretty slim IMO.

Reply:
This code is not edited very often,  so I don't think it's too bad. BTW, I'd really like to use ocamlformat one day, to enforce a uniform formatting.

Reply:
ok!